sdk/python/feast/infra/offline_stores/contrib/contrib_repo_configuration.py

```diff
@@ -1,10 +1,29 @@
 from tests.integration.feature_repos.integration_test_repo_config import (
     IntegrationTestRepoConfig,
 )
+from tests.integration.feature_repos.universal.data_sources.postgres_data_source import (
```

Comment:
nit: name it `postgres.py` to be consistent with the other files (e.g. `file.py`)